requirements.txt

```diff
@@ -10,4 +10,5 @@ spacy >= 2.3.0,<3.0.0
 pandas < 1.3.0 # pinned to this version as higher versions conflicts with swifter version 1.0.5 and higher
 swifter >= 1.0.3
 en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.0/en_core_web_sm-2.3.0.tar.gz
-textstat >= 0.7.0
\ No newline at end of file
+textstat >= 0.7.0
+gingerit == 0.9.0
```

Comment:
Safe way to pin the package version number - any drawbacks to keep it open? The good thing maybe that we know it always worked on this version

Reply:
Yes, so previously, this happened with TextBlob, where their translate function stopped working due to the version upgrade. So, a safe way to go about it is, i believe, to fix the gingerit version to 0.9.0 unless it becomes obsolete.